src/tools/miri/src/bin/miri.rs

```diff
@@ -295,7 +295,7 @@ fn main() {
     // If the environment asks us to actually be rustc, then do that.
     if let Some(crate_kind) = env::var_os("MIRI_BE_RUSTC") {
         // Earliest rustc setup.
-        rustc_driver::install_ice_hook(rustc_driver::DEFAULT_BUG_REPORT_URL, |_| ());
```

Comment:
Why does miri not hide the "report bug" message with internal features?

Reply:
because I couldn't be bothered fixing the tools, will do it later